docs/9-batch-actions.md

```diff
@@ -21,7 +21,7 @@ your desired batch action on all of them:
 ```ruby
 ActiveAdmin.register Post do
   batch_action :flag do |ids|
-    Post.find(ids).each do |post|
+    batch_action_collection.where(id: ids).each do |post|
```

Comment:
we should use `find` like here https://github.com/activeadmin/activeadmin/pull/3775/files#diff-16da2146f502eb423c49bb455a83c2f7R70